lib/extend.js

```diff
@@ -111,18 +113,27 @@ function extend(opts) {
     if (!_.isArray(options.source))
       throw new Error('source must be an array');
 
-    var props = combineJSON(options.source, true, function Collision(opts) {
-      if (options.log) {
-        var str = `Collision detected at: ${opts.path.join('.')}! Original value: ${opts.target[opts.key]}, New value: ${opts.copy[opts.key]}`;
-        if (options.log === 'warn') {
-          console.warn(chalk.keyword('orange')(str));
-        } else if (options.log === 'error') {
-          throw new Error(str);
-        }
+    var ERROR_GROUP = 'Property Value Collisions';
+    var props = combineJSON(options.source, true, function Collision(prop) {
+      var err = new CustomError(
+        `Collision detected at: ${prop.path.join('.')}! Original value: ${prop.target[prop.key]}, New value: ${prop.copy[prop.key]}`,
+        {group: ERROR_GROUP}
+      );
+      ErrorHandler.add(err);
+    });
+
+    if(ErrorHandler.count(ERROR_GROUP) > 0) {
```

Comment:
Here you are using `if(ErrorHandler.count(ERROR_GROUP) > 0) {` and above you are using `if(ErrorHandler.count(ERROR_GROUP)) {`

Reply:
Good catch, thanks!